src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,63 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  currentTime: Date;
+};
+
+function getTimeString(date: Date): string {
+  return date.toUTCString().slice(-12, -4);
+}
+
+export class Clock extends Component<Props, State> {
+  state = {
+    currentTime: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      const newDate = new Date();
```

Comment:
```suggestion
      const newDate = getTimeString(new Date());
```
Use it once instead of repeating here and in render